include/aktualizr-lite/api.h

```diff
@@ -213,6 +233,24 @@ class AkliteClient {
   AkliteClient &operator=(const AkliteClient &) = delete;
   AkliteClient &operator=(AkliteClient &&) = delete;
 
+  /**
+   * @brief Checks whether there is ongoing installation
+   *
+   * Checks whether there is pending installation that has to be completed.
+   * To complete installation a device should be rebooted and/or `CompleteInstallation()` called.
+   * @return true if there is ongoing installation, otherwise false
+   */
+  bool IsInstallationInProgress() const;
+
+  /**
+   * @brief Returns a pending Target if any
+   *
+   * Checks whether there is ongoing installation to be completed and returns corresponding Target.
+   *
+   * @return Pending Target, or "unknown" Target if there is no pending Target
+   */
+  TufTarget GetPendingTarget() const;
```

Comment:
and we don't need any of this in the custom example client because it does the finalization automagically?